packages/terra-date-picker/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Unreleased
 * Added clinical-lowlight-theme theming files
 
 ### Changed
+* DateInput split into 3 distinct inputs
```

Comment:
this need to be moved under the unreleased section

Reply:
Maybe update it as well to say something about how they're masking inputs.